.github/dependabot.yaml

```diff
@@ -0,0 +1,26 @@
+version: 2
+updates:
+- package-ecosystem: "gomod"
+  directory: "/"
+  schedule:
+    interval: "daily"
+  open-pull-requests-limit: 5
```

Comment:
Hey @Neo2308 @JacobOaks I'd like to suggest reconsidering this change.
Adding dependabot for a library means that all users of the library must upgrade to latest version of all transitive dependencies all the time.

Please consider dropping this specific section from the dependabot so that the library's dependencies remain specific to only what it needs when it needs it. Everything else can remain as-is.

Reply:
Thanks @abhinav, you bring up good points. Removed this section with https://github.com/uber-go/mock/pull/163.